s3/pkg/datastore/azure/azure.go

```diff
@@ -46,6 +46,48 @@ type AzureAdapter struct {
 	containerURL azblob.ContainerURL
 }
 
+func (ad *AzureAdapter) BucketDelete(ctx context.Context, input *pb.Bucket) error {
+
+	containerURL, _ := ad.createBucketContainerURL(ad.backend.Access, ad.backend.Security, input.Name)
+
+	_, err := containerURL.Delete(ctx, azblob.ContainerAccessConditions{})
+	if err != nil {
+		log.Error(err)
```

Comment:
why not return from here?

Reply:
will change

Reply:
changed

Reply:
@vineela1999 while logging it will be good if we append/prepend (custom) some messages. This will help in debugging while looking into logs. Basically the context

Reply:
updated